ls1intum/Artemis
# [Bugfix] Fix old master buildplans in Jenkins

### Checklist

### Motivation and Context


The Artemis update 5.0  introduced the option to choose different default branches. This broke the old build plans used in Jenkins that directly refer to the master branch.

### Description

This PR fixes the bug by changing the reference to the master branch whenever a new build plan is created.

### Steps for Testing


1. Log in to Artemis on **ts2**
2. Navigate to an old exercise, that was created before the update to 5.0, for example https://artemistest2.ase.in.tum.de/course-management/615/programming-exercises/4471
3. Check that you can still participate in that exercise as a student
4. Import the old exercise and check that the template and solution build correctly and that you can participate in that exercise as well
